src/Altinn.App.Core/AppImplementationFactory.cs

```diff
@@ -0,0 +1,57 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Altinn.App.Core.Features;
+
+/// <summary>
+/// Marker attribute for interfaces that are meant to be implemented by apps.
+/// </summary>
+[AttributeUsage(AttributeTargets.Interface, AllowMultiple = false)]
+internal sealed class ImplementableByAppsAttribute : Attribute { }
```

Comment:
In this PR just as a mechanism to categorize. I could make a list of interfaces, but then it is hard to know that the list should be updated when an interface is added. There is future potential as well. Main point is having some mechanism to categorize the interfaces (since it is public). Could also be the inverse - public interfaces that are _not_ meant to be public are attributed with `Pubternal` or something